org.aposin.licensescout.configuration.sample/.project

```diff
@@ -5,19 +5,13 @@
 	<projects>
 	</projects>
 	<buildSpec>
-		<buildCommand>
-			<name>org.eclipse.jdt.core.javabuilder</name>
-			<arguments>
-			</arguments>
-		</buildCommand>
 		<buildCommand>
 			<name>org.eclipse.m2e.core.maven2Builder</name>
 			<arguments>
 			</arguments>
 		</buildCommand>
 	</buildSpec>
 	<natures>
-		<nature>org.eclipse.jdt.core.javanature</nature>
```

Comment:
Is really needed to remove the java nature?

Reply:
We should do so because the project does not contain Java source code. The bundling works with a custom configured packaging, not with the maven-jar-plugin - therefore the maven packaging type of the project is 'pom' (for normal Java projects it is 'jar').

The starting point to investigate this was that somebody mocked about the outdated Java runtime specification in the .classpath file. Instead of maintaining it, it is better to remove it completely (and therefore other definitions related to it) to reflect the true nature of the project. So removing is is the clean solution that saves maintenance time.